src/modules/launcher/Wox.Plugin/Common/DefaultBrowserInfo.cs

```diff
@@ -127,6 +127,14 @@ public static void Update()
                         commandPattern = commandPattern.Insert(0, "\"");
                     }
 
+                    // Arc.exe takes no parameters.
+                    // Give up and use $"rundll32 url.dll,FileProtocolHandler {URL}" instead.
+                    const string ArcExecutableName = "Arc.exe";
+                    if (commandPattern.Contains(ArcExecutableName))
+                    {
+                        commandPattern = "rundll32 url.dll,FileProtocolHandler"
```

Comment:
```suggestion
                        commandPattern = "rundll32 url.dll,FileProtocolHandler";
```

@TheRedDeveloper 
I think this is why the code does not build. The CI definitely complains about that.